parse-cabal-project.nix

```diff
@@ -0,0 +1,15 @@
+cabalProject:
+let
+  content = builtins.readFile cabalProject;
+  lines = builtins.filter builtins.isString (builtins.split ''
+    [
+    ]'' content);
+  matches =
+    builtins.map (builtins.match "[[:space:]]*[.]/(.*)/(.*)[.]cabal$") lines;
+  projects = builtins.concatMap (match:
+    if builtins.isList match && builtins.length match == 2 then [{
+      name = builtins.elemAt match 1;
+      path = builtins.elemAt match 0;
+    }] else
+      [ ]) matches;
+in projects
```

Comment:
Could you add a comment to this referencing NixOS/cabal2nix#286, or, even better, submit this there in a PR and link to that.

Reply:
added the reference there.